packages/ui/src/walkthrough-card/WalkthroughCard.css

```diff
@@ -0,0 +1,46 @@
+.ui-WalkthroughCard--CardContainer {
+  display: flex;
+  border-radius: var(--layout-border-radius-xlarge);
+  box-shadow: var(--surface-shadow-none), var(--surface-shadow-none);
+  background-color: var(--surface-color-background-white);
+  max-width: var(--walkthrough-card-max-width);
+  flex-direction: column;
+  font-size: 14px;
```

Comment:
Do we need this font declaration here? I think all the content inside has it's own font declarations right? Same with the color line below this.